hive_mind_hive/app/assets/javascripts/hive_mind_hive/load_graph.js

```diff
@@ -9,13 +9,13 @@ load_graph = function(svg_class, dataset, width, height) {
     .enter()
     .append('rect')
     .attr('x', function(d,i) {return i * bar_width })
-    .attr('y', function(d) { return height - d*height })
+    .attr('y', function(d) { return height - d*height/cpu_count })
     .attr('width', bar_width)
-    .attr('height', function(d) {return d*height})
+    .attr('height', function(d) {return d*height/cpu_count})
     .attr('fill', function(d) {
-      if (d>=1.5) {
+      if (d/cpu_count>=1.5) {
```

Comment:
Could we have the 1.5 and 0.7 as config items? Given they're arbitrary values it should be easy to change them